### ISSUE TITLE
A importable chat component for firebase projects.

### ISSUE THREAD

**Issue by joshuaferr1s** [CodingGarden/landscaping#11]
## (REQUIRED) What category does this issue fit into?

>Please limit your issue to 1 category.

* [ ] App Idea
* [ ] App Constraint
* [x] Front-end Technology
* [x] Back-end Technology
* [ ] Deployment
* [ ] DevOps
* [ ] Code Craft
* [ ] Workflow/Tools
* [ ] Show Type or Series
* [ ] Something Else

## (REQUIRED) How are you affiliated with Coding Garden?

Please check all that apply

* [x] [YouTube Subscriber](https://www.youtube.com/codinggardenwithcj?sub_confirmation=1)
* [ ] [Patron](https://www.patreon.com/CodingGardenWithCJ)
* [ ] [Donor](https://streamlabs.com/CodingGardenWithCJ)

## (REQUIRED) Describe your suggestion

A chat component (Vue or otherwise) that can be imported into an existing firebase project.

**Comment by w3cj** [comment]
Interesting! Would the idea be this is an embed-able chat box component? Like drop it onto a page and it connects to the existing firebase connection on the page, and logs chats there?

**Comment by joshuaferr1s** [comment]
Exactly! So all you need is firebase to already be configured and theoretically you can just import it onto the page you want it.

**Comment by hermanocabral** [comment]
like this? https://github.com/berksaribas/vuetify-chat?ref=madewithvuejs.com

**Comment by joshuaferr1s** [comment]
Essentially
